procfiles/run_devnet_cluster_demo.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
```

Comment:
Uhm, well, I'm not sure I want those guys in version control just yet. Maybe a separate PR?

Reply:
yup, this is why I have a separate, standalone commit for that 😬

Reply:
d3378c5